.github/workflows/integration-tests.yml

```diff
@@ -13,17 +13,15 @@ permissions:
 
 jobs:
   run_tests:
-    runs-on: ubuntu-latest
+    runs-on: github-hosted-ubuntu-x64-large
     steps:
     - name: Checkout code
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       with:
         persist-credentials: false
     - name: Setup Go
-      uses: actions/setup-go@0aaccfd150d50ccaeb58ebd88d36e91967a5f35b # v5.4.0
+      uses: actions/setup-go@v5
```

Comment:
Why was this changed? We should keep actions pinned to sha for security reasons.